File: validate_phone.py
def validate_phone(phone):
    import re
    #phone = "0557766915"
    ##Telephone portable
    ##Telephone fixe
    regexp1 = r"^0[0-9]([ .-]?[0-9]{2}){4}$"
    regexp2 = r"^0([ .-]?[0-9]{2}){4}$"
    if re.match(regexp1, phone) or re.match(regexp2, phone):
        return True
    else:
        return False

def main():
    print(validate_phone("0557766915"))

if __name__ == '__main__':
    main()

File: validate_phone.js
function VerifierTel(){
    // var regex = /^(00213|\+213|0)(5|6|7)[0-9]{8}$/;
    var regexp1 = /^0(5|6|7)[0-9]{8}$/;
    var regexp2 = /^0[0-9]{4}$/;

    var tel = document.forms["FormConfirm"]["phone"].value;
    console.log("tel : "+ tel)
    console.log(" len (tel) : "+ tel.length)
    if(!regexp1.test(tel) && !regexp2.test(tel))
    {
      alert("un numero de telephone doit avoir 7 chiffres pour le fixe et 10 chiffres pour le mobile");
      document.forms["FormConfirm"]["phone"].style.value ="";
      document.forms["FormConfirm"]["phone"].style.border="red";
      return false;
    }
    else
    {
      return true;
    }
}
